x-pack/plugins/transform/common/types/transform_stats.ts

```diff
@@ -17,7 +17,12 @@ export interface TransformHealthIssue {
   count: number;
   first_occurrence?: number;
 }
-
+export type FetchStatus = 'error' | 'success' | 'loading';
+export enum FETCH_STATUS {
+  ERROR = 'error',
+  SUCCESS = 'success',
+  LOADING = 'loading',
+}
```

Comment:
Nit: In most other places we define this with the `as const` pattern and then derive the union type from it, for example like here with `TRANSFORM_STATE/TransformState`: https://github.com/elastic/kibana/blob/main/x-pack/plugins/transform/common/constants.ts#L92

Reply:
This is now reverted due to latest change